C/cp/doublefactorial.c

```diff
@@ -0,0 +1,28 @@
+#Double factorial of a non-negative integer n is defined as the product of all the integers from 1 to n that have the same parity (odd or even) as n.
```

Comment:
@bagladivyang03 This is not the correct way for commenting in C. Please use multi-line comment here `/* */`